### DISCUSSION TITLE
New feed

### DISCUSSION THREAD

**Discussion by odenirdev** [community/community#13141] [Other Feature Feedback, Questions, & Ideas]
I thought the new feed is very useful, bringing suggestions about new devs and projects to follow and interactions about repos i watching.

**Comment by euhidaman** [comment]
I like the fact, that it's now showing me my own repetitive GitHub Actions workplay in my feed continuously.

**Comment by jfullstackdev** [comment]
much more,

there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
